esp/src/eclwatch/_Widget.js

```diff
@@ -130,22 +130,22 @@ define([
             return retVal;
         },
 
-        getURL: function (filterFromComponent) {
+        getURL: function () {
             var baseUrl = document.URL.split("#")[0];
             var baseUrlParts = baseUrl.split("?");
             baseUrl = baseUrlParts[0];
             var args = baseUrlParts[1];
             delete this.params.__filter;
-            var filterParams = filterFromComponent ? filterFromComponent : this.getFilterParams();
+            var filterParams = this.getFilterParams();
```

Comment:
`var filterParams = !filterFromComponent ? this.getFilterParams() : filterFromComponent;` ?